workflows/raredisease.nf

```diff
@@ -32,6 +31,14 @@ if (params.run_rtgvcfeval) {
     mandatoryParams += ["rtg_truthvcfs"]
 }
 
+if (!params.skip_repeat_analysis) {
+    mandatoryParams += ["variant_catalog"]
+}
+
+if (!params.skip_snv_calling) {
+    mandatoryParams += ["genome"]
```

Comment:
`genome` is also used by smncopynumbercaller, but maybe you're adding that in a later PR :)

Reply:
Yupe! I have a couple more PR's in line to be merged after this :D